app_dart/index.yaml

```diff
@@ -13,6 +13,11 @@ indexes:
   - name: Branch
   - name: CreateTimestamp
     direction: desc
+- kind: Checklist
```

Comment:
@CaseyHillers based on the prospective queries we will want to make, should we be adding the repo info to Task entities directly, versus using the property in Checklist?

Reply:
It isn't necessary for get-status. It grabs tasks if they have the parent key of the checklist - https://cs.opensource.google/flutter/cocoon/+/869becfbe169ce10c561f507c8984e2233e1be51:app_dart/lib/src/service/datastore.dart;l=124